app/models/hackbot/conversations/check_in.rb

```diff
@@ -143,6 +151,10 @@ def wait_for_notes(event)
 
       private
 
+      def record_notes(event)
+        data['notes'] = event[:text] unless event[:text] =~ /^(no|nope|nah)$/i
```

Comment:
Can you think of a better name for this method than `record_notes`? `record_notes` sounds like it's just recording notes from a given parameter, not extracting notes from an event, checking to see the user didn't say no, and then recording the given notes.

Perhaps a problem of separation of concerns?